src/API.jl

```diff
@@ -772,6 +775,7 @@ status(pkgs::Vector{<:AbstractString}; kwargs...) =
 status(pkgs::Vector{PackageSpec}; kwargs...) = status(Context(), pkgs; kwargs...)
 function status(ctx::Context, pkgs::Vector{PackageSpec}; diff::Bool=false, mode=PKGMODE_PROJECT,
                 kwargs...)
```

Comment:
```suggestion
                io::IO=stdout, kwargs...)
        Context!(ctx; io=io, kwargs...)
```

So the user can still override if necessary

Reply:
Should be equivalent to what I have now. If a user has `io` as a kwarg it will be overwritten at L779.